Rect42.dart

```diff
@@ -0,0 +1,121 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+
+class Rect42 extends StatefulWidget {
+  final int count;
+
+  const Rect42({Key? key, required this.count}) : super(key: key);
+
+  @override
+  State<Rect42> createState() => _Rect42State();
+}
+
+class ListColour<listRow> {

```

Comment:
no need to have a separate class for that, internal class function could me more than enough